.github/labeler.yml

```diff
@@ -45,32 +28,24 @@ INFRA:
   - "dev/merge_spark_pr.py"
   - "dev/run-tests-jenkins*"
 BUILD:
- # Can be supported when a stable release with correct all/any is released
- #- any: ['dev/**/*', '!dev/merge_spark_pr.py', '!dev/.rat-excludes']
- - "dev/**/*"
```

Comment:
The changes to the file `dev/.rat-excludes` should also belong to the label `BUILD`, So we shouldn't exclude it out here.